engine/src/test/java/org/hibernate/search/test/searchfactory/SearchFactoryTest.java

```diff
@@ -49,6 +67,158 @@ public void testTypeWithNoDocumentIdThrowsException() {
 	}
 
 	@Test
+	@Category(ElasticsearchSupportInProgress.class) // HSEARCH-2481 Byteman-based tests re-executed in the Elasticsearch module won't work
+	@TestForIssue(jiraKey = "HSEARCH-2277")
+	@BMRules(rules = {
+			@BMRule(
+					name = "Skip failure on first index manager initialization",
```

Comment:
What's the need for this rule? It doesn't look like you need this Flag, you can just omit the condition in the second rule?